.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+    branches: [ $default-branch ]
+  release:
+    types:
+      - created
+
+jobs:
+  test:
+    name: CI
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        otp: [21.3.8.18, 22.0.7, 22.1.8, 22.2.8, 22.3.4.14, 23.0.4, 23.1.5.0, 23.2.1.0]
+    container:
+      image: erlang:${{ matrix.otp }}-alpine
+    steps:
+    - uses: actions/checkout@v2
+    - name: Prepare
+      run: |
+           apk update
+           apk --no-cache upgrade
+           apk --no-cache add gcc git libc-dev libc-utils libgcc linux-headers make bash musl-dev musl-utils ncurses-dev pcre2 pkgconf scanelf wget zlib
+    - name: Build
+      run: rebar3 compile
+    - name: Run tests
+      run: |
+           rebar3 do xref
+           rebar3 do ct
+           rebar3 as test do coveralls send || /bin/true
```

Comment:
coverall on GH actions requires rebar.config.script changes. Also, running a matrix test requires the coverall parallel option and the end webhook

Reply:
Done